packages/framework/memo-cache/CHANGELOG.md

```diff
@@ -0,0 +1,128 @@
+# Change Log - @uifabricshared/immutable-merge
```

Comment:
Looks like the wrong changelog 😊, I think the CHANGELOG.json has the same problem.

Reply:
Doh! Obviously started this package by just copying immutable merge and modifying it :)